contrib/completion/zsh/_docker

```diff
@@ -78,7 +78,7 @@ __docker_get_containers() {
             s="${${line[${begin[CONTAINER ID]},${end[CONTAINER ID]}]%% ##}[0,12]}"
             s="$s:${(l:15:: :::)${${line[${begin[CREATED]},${end[CREATED]}]/ ago/}%% ##}}"
             s="$s, ${${${line[${begin[IMAGE]},${end[IMAGE]}]}/:/\\:}%% ##}"
-            if [[ ${line[${begin[STATUS]},${end[STATUS]}]} = Exit* ]]; then
+            if [[ ${line[${begin[STATUS]},${end[STATUS]}]} = (Exit*|Created*) ]] ; then
```

Comment:
I'm gonna be very nit-picky, but can you remove the extra space before the semicolon, to keep a consistent coding style in this file? 😅